tests/test_oml/test_functional/test_metrics/test_cmc_metric_old.py

```diff
@@ -38,26 +38,27 @@ def cmc_score(
 
 TEST_DATA_SIMPLE = (
     # (distance_matrix, mask_gt,  topk, expected_value)
-    (torch.tensor([[1, 2], [2, 1]]), torch.tensor([[0, 1], [1, 0]]), 1, 0.0),
-    (torch.tensor([[0, 0.5], [0.0, 0.5]]), torch.tensor([[0, 1], [1, 0]]), 1, 0.5),
-    (torch.tensor([[0, 0.5], [0.0, 0.5]]), torch.tensor([[0, 1], [1, 0]]), 2, 1),
+    (torch.tensor([[1, 2], [2, 1]]), torch.tensor([[0, 1], [1, 0]], dtype=torch.bool), 1, 0.0),
```

Comment:
I think that changing to bool is not related to the current PR. Let's roll back this file.

Reply:
There was an error in `test_metric_greater` because `cmc_score_count` calls `calc_retrieval_metrics` and `fmr_vals` was set to default value `(1.e-3, )`. This made `calc_retrieval_metrics` call `calc_fnmr_at_fmr`, which in its turn had to evaluate `distances[mask_gt]` and pytorch threw the error because `mask_gt` is not int or bool type. So I will roll back this file, but I'll set `fmr_vals` to `tuple()` in `cmc_score_count` to supress `fnmr@fmr` evaluation.